### ISSUE TITLE
Use existing translated strings while possible

### ISSUE THREAD

**Issue by overflorian** [maproulette/maproulette3#2380]
Hi, Transifex suggests me to translate the very basic word "Challenge" (see https://app.transifex.com/osmlab/maproulette3/translate/#fr/en-us-json/476126110) even if it has already been translated before (see https://app.transifex.com/osmlab/maproulette3/translate/#fr/en-us-json/170937424).

I suggest that you re-use the same translation string when exactly the same, avoiding the interface not being fully translated for no reason + avoid some translators time too.

**Comment by mvexel** [comment]
I agree that seems silly. Moreover, we already have `challenge` as [a glossary term](https://app.transifex.com/osmlab/maproulette3_2020-05-24/translate/#fr/maproulette3_2020-05-24/432463400) already. What would be involved in streamlining this @jschwarz2030 @CollinBeczak ?

**Comment by CollinBeczak** [comment]
This can be done relatively easily, all of these duplicate terms have unique ids, these ids are usually related to the feature they are associated with. So if we were to take the approach as described above and remove duplicates, we would probably want to replace all those with a new id that is not tied to a particular context like `id: "challenge"` instead of `id: "Inbox.tableHeaders.challengeName".` The only annoyance i see with this from a translator perspective is that they will need to re translate all the strings that had duplicates.

**Comment by mvexel** [comment]
My sense is that the benefits outweigh that inconvenience, but I can't quite assess the impact on translators. Let's keep this open for a bit and give others a chance to weigh in.

**Comment by CollinBeczak** [comment]
If we choose to take this route using existing translated IDs, it should have no impact on translators—unless the IDs hold additional significance that translators rely on.
